### ISSUE TITLE
Exception: Missing target values in the training set after data split: {32, 33, 36, 37, 38, 39}.

### ISSUE THREAD

**Issue by esitharth** [alteryx/evalml#2542]
Hi I am getting this error when I run automl.search().

I have even tried dropping NA Values from the target column before the split.

**Comment by freddyaboulton** [comment]
Hi @esitharth ! Thanks for filing an issue! Can you post a reproducible example. That error arises when there's a classification problem and some of the labels are only present in the validation set. That usually happens when there's extreme class imbalance.
